buildifier/runner.bat.template

```diff
@@ -0,0 +1,45 @@
+@echo off
+
+set ARGS=@@ARGS@@
+
+rem Remove the leading '(' and trailing ')' characters that surround the arguments
+set stripped_args=%ARGS:~1,-1%
+
+rem Unquote the arguments
+set stripped_args=%stripped_args:'=%
+
+rem Get the absolute path to the buildifier executable
+for /f "tokens=2" %%i in ('findstr /r "\<buildifier\.exe\>" MANIFEST') do (set buildifier_abs_path=%%i)
+
+powershell ^
+function Buildify($Root)^
```

Comment:
Buildifier has the `-r` flag to recursively find all relevant files within a directory, maybe it makes sense to reuse it? The built-in recursive function can ignore some paths (currently just `.git`, but may support `.buildifierignore` in the future, see #801), I think it's better to have such logic defined once.

Reply:
But the Linux version does it recursively by default, without needing the `-r` flag. I think the Windows and Linux versions should have the same behavior.

Reply:
Maybe it had been implemented this way before the `-r` flag was introduced. I can fix it later, but won't be able to fix the windows version easily because don't have a windows machine around. So I suggest using `-r` from the beginng.

Reply:
How do I use the `-r` flag with `bazel run //:buildifier`? I'm able to use it with the standalone executable but not as a bazel rule.

Reply:
I noted on https://github.com/bazelbuild/buildtools/issues/1136 that the `-r` flag also does not match the same set of files.  The set used by the runner template is more complete.  Ideally, those should probably be aligned, but I'd be in favor of merging this in for running on Windows first.